tests/testthat/test-map_region_risk.R

```diff
@@ -1,27 +1,86 @@
 test_that("returns an object of the expected class", {
-  try({
-    plot <- map_region_risk(without_financial)
-  })
+  skip_on_ci()
+  data <- example_without_financial(
+    postcode = 53773L,
+    !!aka("emission_profile") := c("high", "medium", "low")
+  )
+  plot <- map_region_risk(data)
   expect_s3_class(plot, "ggplot")
 })
```

Comment:
`map_region_risk()` is terribly slow. You may find a way to mock the call to the external API (assuming that's the bottleneck). But in the meantime you may want to minimize the number of tests you run on this function -- by removing redundant tests like this one. Don't worry, if your function doesn't return a ggplot object, your other tests will let you know.

<details/>
<summary/>
reprex
</summary>

``` r
devtools::load_all()
#> ℹ Loading tiltPlot

# Relatively fast
system.time(plot_sankey(financial))
#>    user  system elapsed 
#>   0.051   0.000   0.051
# Relatively slow (actually very slow)
system.time(map_region_risk(without_financial))
#> Extracting data using giscoR package, please report issues on https://github.com/rOpenGov/giscoR/issues
#> Cache management as per giscoR. see 'giscoR::gisco_get_nuts()'
#>    user  system elapsed 
#>   1.137   0.081  14.600
```

</details>